src/Lykke.Service.HFT.Services/MatchingEngineAdapter.cs

```diff
@@ -85,10 +121,16 @@ public MatchingEngineAdapter(IMatchingEngineClient matchingEngineClient, IReposi
             return ConvertToApiModel(response.Status);
         }
 
-        public async Task<ResponseModel> TransferAsync(string fromClientId, string toClientId, string assetId, double amount)
+        public async Task<ResponseModel> TransferAsync(string fromClientId, string toClientId, string assetPairId, double amount)
         {
             var id = GetNextRequestId();
-            var response = await _matchingEngineClient.TransferAsync(id, fromClientId, toClientId, assetId, amount);
+
+            var assetPair = await _assetsService.AssetPairGetAsync(assetPairId);
+            var fee = await _feeCalculatorClient.GetMarketOrderFees(fromClientId, assetPairId, assetPair.BaseAssetId, FeeCalculator.AutorestClient.Models.OrderAction.Sell);
+
+            var response = await _matchingEngineClient.TransferAsync(id, fromClientId, toClientId,
+                assetPair.BaseAssetId, amount, _feesSettings.TargetClientId, (double) fee.DefaultFeeSize);
```

Comment:
I have some doubts of this code, but it is not using anywhere, I'll just delete it in next commit.